pom.xml

```diff
@@ -156,7 +156,7 @@
         <air.check.skip-spotbugs>true</air.check.skip-spotbugs>
         <air.check.skip-pmd>true</air.check.skip-pmd>
         <air.check.skip-jacoco>true</air.check.skip-jacoco>
-        <air.java.version>23.0.0</air.java.version>
+        <air.java.version>23</air.java.version>
```

Comment:
switch this to 24 as well?

Reply:
No. This will come next (hard requirement)